features/__snapshots__/task_validate_image.snap

```diff
@@ -69,13 +69,26 @@ components:
       code: builtin.attestation.syntax_check
       title: Attestation syntax check passed
     msg: Pass
+  - metadata:
+      code: builtin.image.config_check
+      title: Image config is accessible
+    msg: Pass
   - metadata:
       code: builtin.image.signature_check
       title: Image signature check passed
     msg: Pass
   warnings:
+  - metadata:
+      code: builtin.parent_image.config_check
+      title: Parent image config is accessible
+    msg: 'Parent image config is not accessible: unable to fetch parent image: Get
+      "https://localhost:37957/v2/": dial tcp [::1]:37957: connect: connection refused;
+      Get "http://localhost:37957/v2/": dial tcp [::1]:37957: connect: connection
```

Comment:
Unclear what was happening here. With `-persist`, I was able to query the registry just fine from the terminal. Not sure if there was a timing issue during the tests. For now, this is no longer an issue because I removed the newly added `builtin` results.